packages/core/src/types/vc-data-model.ts

```diff
@@ -44,14 +44,29 @@ export type CredentialSubject = {
  *
  * @beta This API may change without a BREAKING CHANGE notice.
  */
-export type CredentialStatus = {
+export type CredentialStatusReference = {
   id: string
   type: string
   [x: string]: any
 }
 
 /**
- * A proof property of a Verifiable Credential or Presentation
+ * Represents the result of a status check.
+ *
+ * Implementations MUST populate the `revoked` boolean property, but they can return additional metadata that is
+ * method specific.
+ *
+ * @see {@link credential-status#CredentialStatus | CredentialStatus}
+ *
+ * @beta This API may change without a BREAKING CHANGE notice.
+ */
+export type CredentialStatus = {
+  revoked?: boolean
```

Comment:
If implementations MUST populate `revoked`, we shouldn't mark it as optional, right?

Reply:
```suggestion
  revoked: boolean
```

Reply:
You are right. It has to be non-optional.

Reply:
@italobb I made the `revoked` property required and added a check for the `StatusMethod` returned value to look for the `revoked` property.
An error is thrown if the property is missing, and I adjusted the tests to include this.

Reply:
Great! Thanks.